memory/cloudsql/chat_message_history.go

```diff
@@ -105,3 +108,65 @@ func (c *ChatMessageHistory) validateTable(ctx context.Context) error {
 	}
 	return nil
 }
+
+// addMessage adds a new message into the ChatMessageHistory for a given
+// session.
+func (c *ChatMessageHistory) addMessage(ctx context.Context, content string, messageType llms.ChatMessageType) error {
+	data, err := json.Marshal(content)
```

Comment:
Isn't content a string? Why does it need to be marshaled?

Reply:
yes content is a string, Averi left a comment on AlloyDB's PR that data type is JSON in python canonical https://github.com/googleapis/langchain-google-cloud-sql-pg-python/blob/main/src/langchain_google_cloud_sql_pg/async_chat_message_history.py#L97.